data/mods/blazemod/recipes/vortex_recipes.json

```diff
@@ -0,0 +1,142 @@
+[
```

Comment:
Vortex stuff is strange, I almost had duplicates in here but that's sorted. Otherwise, unchanged.